### DISCUSSION TITLE
Clicking on calculated eclipse data changes location?

### DISCUSSION THREAD

**Discussion by Keatah** [Stellarium/stellarium#2584] [Q&A] [answered]
Is there a way to set Stellarium to stay at the present location when clicking on the eclipse data row in the eclipses tab in AstroCalc window?

Maybe there could be an option made to reset the location to what it was. Like a back button. Or have hotkeys for pre-defined locations. Or have a list of pre-defined locations that we can cycle through.

**Comment by gzotti** [comment]
There are two tabs: All solar eclipses, where clicking on the eclipse takes you to the point of best visibility, and local eclipses, which compute local data.

**Comment by Keatah** [comment]
Yes. Thanks for that. It works.

I did notice a what seems to be a minor bug. The red arrow on the location map seems to change its position only **one time** when picking eclipses from the table, even when clicking on "All Solar Eclipses" and selecting those.

I can recalculate or just pick another eclipse, but the arrow doesn't move to the new location. Though the main view seems to change/move as expected.

**Reply by gzotti** [reply]
Uh, that should be fixed of course. Maybe just some signal connection missing? @alex-w ?

**Reply by alex-w** [reply]
Fixed

**Comment by alex-w** [comment, marked as answer]
> Maybe there could be an option made to reset the location to what it was. Like a back button. Or have hotkeys for pre-defined locations.

`Ctrl+H`

**Comment by Keatah** [comment]
Thanks.

I must remember to check the well-written pdf manual and key assignments list more often. Listed in Appendix A Default Hot Keys and in-program. There's so much good material in there.
